maps/expert/dark_castle/chimps.ahk

```diff
@@ -0,0 +1,135 @@
+DarkCastleChimps() {
+    global TS := Map(
+        "Obyn",["hero",[865,447]],
+        "Dart A",["dart",[739,656]],
+        "Dart B",["dart",[688,691]],
+        "Sub A",["sub",[1083,690]],
+        "Sub B",["sub",[1086,398]],
+        "Alch A",["alch",[1011,657]],
+        "Alch B",["alch",[666,388]],
+        "Alch C",["alch",[711,346]],
+        "Druid A",["druid",[715,433]],
+        "Druid B",["druid",[789,447]],
+        "Druid C",["druid",[941,448]],
+        "Druid D",["druid",[766,384]],
+        "Druid E",["druid",[842,384]],
+        "Druid F",["druid",[918,385]],
+        "Spike",["spike",[1022,441]],
+        "Village",["village",[869,303]]
+    )

```

Comment:
I would like to see a space after every comma like in every other game script file. Some parts of the code are still missing these spaces but it would be nice if at least all game scripts were correctly formatted. And even though I only highlighted this map, I'd like to see every function call be like this also.

Other scripts also have the towers listed in the order they are placed in but that doesn't really matter and this does look better so no need to change that.

Reply:
Fixed the commas, and will keep this in mind for future scripts.
I won't change the order for now, can do that in the future though.